hw/ip/prim/dv/prim_ascon/ascon_model_dpi/vendor/ascon_ascon-c.vendor.hjson

```diff
@@ -0,0 +1,28 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "ascon_ascon-c",
+  target_dir: "ascon_ascon-c",
+
+  upstream: {
+    url: "https://github.com/ascon/ascon-c.git",
+    rev: "main",
+  },
+
+  mapping : [
+    {from: 'crypto_aead/ascon128v12/ref', to: 'ascon128'},
+    {from: './tests/crypto_aead.h', to: 'ascon128'}
```

Comment:
Nit: Does this actually need the "./" prefix? I don't *think* so (but wrote the implementation a long time ago!). If it's not necessary, maybe better to drop it.

Reply:
According to https://opentitan.org/book/util/doc/vendor.html it shouldn't be needed. I'll test it.